src/annotator/anchoring/test/fake-pdf-viewer-application.js

```diff
@@ -129,6 +129,11 @@ class FakePDFViewer {
     this.viewer = this._container;
 
     this.eventBus = new EventEmitter();
+
+    /** @type {'auto'|'page-fit'|'page-width'} */
+    this.currentScaleValue = 'auto';
+
+    this.update = sinon.stub();
```

Comment:
These are extensions to the `PDFViewerApplication` implementation are used by `PDF#fitSideBySide`.